src/redux/apiSlice.ts

```diff
@@ -34,7 +34,8 @@ const baseQueryWithReauth: BaseQueryFn<string | FetchArgs, unknown, FetchBaseQue
 
   if (result.error && result.error.status === RESULT_ERROR_STATUS) {
     // try to get a new token
-    console.log('sending refresh token....');
+    // eslint-disable-next-line no-console
+    console.log('sending refresh token....'); // NOSONAR
```

Comment:
are these consoles really necessary

Reply:
This is not needed. This can be removed.